ci/_init.sh

```diff
@@ -15,20 +15,25 @@ fi
 export DRIVER_NAME=nodejs
 
 # Build images
-BUILD_IMAGE_VERSION=2
+BUILD_IMAGE_VERSION=1
 
 # Test Images
-TEST_IMAGE_VERSION=2
+TEST_IMAGE_VERSION=1
 
 declare -A BUILD_IMAGE_NAMES=(
-    [$DRIVER_NAME-centos7-node14]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-centos7-node14-build:$BUILD_IMAGE_VERSION
+    [$DRIVER_NAME-chainguard-node18]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-chainguard-node18-build:$BUILD_IMAGE_VERSION
 )
 export BUILD_IMAGE_NAMES
 
 declare -A TEST_IMAGE_NAMES=(
-    [$DRIVER_NAME-centos7-fips]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-centos7-fips-test:$BUILD_IMAGE_VERSION
-    [$DRIVER_NAME-centos7-node14]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-centos7-node14-test:$BUILD_IMAGE_VERSION
-
+    [$DRIVER_NAME-chainguard-node18]=$DOCKER_REGISTRY_NAME/client-$DRIVER_NAME-chainguard-node18-test:$TEST_IMAGE_VERSION
```

Comment:
Do we need both build and test images? Maybe we can get rid of build images?

Reply:
I think it was done this way, because we have one build image (node18) and two tests images (node18, node18-fips) and only want to build and upload artifacts to s3 once. We can add some logic to instead choose one of the tests images for build, but I'm not sure it will be more readable. @sfc-gh-pmotacki WDYT ?